### ISSUE TITLE
Wrong password : Arch

### ISSUE THREAD

**Issue by XeGrox** [CypherpunkArmory/UserLAnd#577]
When I enter the password it is always wrong, although I am very sure I entered the correct password and even tried recreating the file system with a different password 3 times. I have also tried 'userland'...

This seems to only affect arch

**Comment by corbinlc** [comment]
In general Arch works for people, so we are going to need some more info to understand what is going wrong.  To start, can you update your issue to follow this template...

https://github.com/CypherpunkArmory/UserLAnd/issues/new?template=bug_report.md&title=Bug+report+-+%5BYOUR+BUG+TITLE%5D

That template includes some info that we often find helpful when debugging.

**Comment by XeGrox** [comment]
Create a new app and fill in the password blank.
Attempt to login with the same password in ssh login
SSH prompts your password again and again no matter what you type

Emui 9.0 (android 9.0)
Userland v2.2.3

I have tried reinstalling, clearing support files and clearing data but to no avail

Seems to only affect the arch distrubution

**Comment by MatthewTighe** [comment]
I can't reproduce this. If you aren't entering your password incorrectly (which doesn't seem to be the case), my only guess is that the password you're trying to use is somehow not formed correctly. This could potentially be something like weird characters, or a length that isn't suitable. If that were the case, we can update our validation of ssh passwords to guard against it.

@corbinlc Any insight into this?

**Comment by EnderNightLord-ChromeBook** [comment]
did you try uninstalling? Did you try older versions?

**Comment by XeGrox** [comment]
@MatthewTighe Yep I have tried using numbers/letters/longer/shorter but it still fails

@EnderNightLord-ChromeBook I don't think older versions would help since the file system is downloaded online but I have tried reinstalling.

**Comment by corbinlc** [comment]
@XeGrox the only interesting thing I can think of is that you are one of the first users of EMUI 9.  EMUI 9 solves a lot of issues we had with Huawei devices previously, but maybe there are still some oddities that are Huawei specific.  Can you try to capture a proot log of you trying to log in?  Here is how to do that...
1) stop any running app/distro
2) in the app settings, enable proot debugging and turn the proot debug verbosity to 9
3) start arch distro again and try to log in a couple of times and seeing it fail
4) send the file, normally found at /sdcard/PRoot_Debug_Log to us
There should be some interesting breadcrumb in that log.
Thanks,
Corbin

**Comment by MatthewTighe** [comment]
@XeGrox When you were trying to run the session that caused that log, was it the same password error? The log makes it look like we probably failed to download an asset, the `extractFilesystem` script we use to expand the filesystem. 

Can you clear support assets using the main menu and provide another log to use afterwards?

**Comment by corbinlc** [comment]
Sorry I didn't notice there was already a PRoot log.

**Comment by MatthewTighe** [comment]
He added it in an edit in response to your comment.

**Comment by XeGrox** [comment]
@corbinlc https://pastebin.com/8RK3GzEv

Sorry for the confusion, the previous log was about the app crash

**Comment by XeGrox** [comment]
Is seems that after some trial and error, the username must be set to `user` for the password to be set correctly. I'm now able to login with username as user

**Comment by EnderNightLord-ChromeBook** [comment]
Horray! Thats Great!

**Comment by XeGrox** [comment]
It seems that the problem was because my username started with a capital letter (XenRon) and prevented the user from being created so I could not log in.
